### PR TITLE
Enable new single_facilitator_experiment type

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: dashboard/app/models/single_facilitator_experiment.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#
+# Indexes
+#
+#  index_experiments_on_max_user_id           (max_user_id)
+#  index_experiments_on_min_user_id           (min_user_id)
+#  index_experiments_on_overflow_max_user_id  (overflow_max_user_id)
+#  index_experiments_on_section_id            (section_id)
+#
+
+class SingleFacilitatorExperiment < Experiment
```
Comment (balderdash) [inline dashboard/app/models/single_facilitator_experiment.rb]: I don't see anything here that's unique to facilitators. Unless you're planning to add some special facilitator logic later, could you rename this to `SingleUserExperiment`?

Comment (mehalshah) [reply dashboard/app/models/single_facilitator_experiment.rb]: Sure thing

File: dashboard/test/models/experiment_test.rb [hunk 2]
```diff
@@ -94,6 +94,17 @@ class ExperimentTest < ActiveSupport::TestCase
       section_id: section.id + 1
     assert_empty Experiment.get_all_enabled(section: section)
   end
+
+  test 'single user experiment is enabled' do
+    facilitator_yes = User.first || create(:facilitator)
+    facilitator_no = User.second || create(:facilitator)
+    experiment = SingleUserExperiment.first || create(:single_user_experiment, min_user_id: facilitator_yes.id)
```
Comment (aoby) [inline dashboard/test/models/experiment_test.rb:101]: @mehalshah Why this first || create pattern? Shouldn't this test, like other dashboard tests, always create the objects it cares about?

Comment (ashercodeorg) [reply dashboard/test/models/experiment_test.rb:101]: Also, why are we creating facilitators? Doing so suggests to me that somehow the test should depend on the user being a facilitator (which it seemingly does not).

**Pull request merged**